### ISSUE TITLE
Stein Variational Gradient Descent

### ISSUE THREAD

**Issue by shoyer** [blei-lab/edward#372]
https://arxiv.org/abs/1608.04471

It looks like an interesting inference technique that would be nice to have in Edward.

**Comment by dustinvtran** [comment]
yeah that would be great to have. we have operator vi in a separate code base (https://arxiv.org/abs/1610.09033), which is very related; we're hoping to merge operator vi in soon. in my experience i haven't found stein variational gradient descent to scale well to high dimensions (due to the kernel).

**Comment by OlaRonning** [comment]
Is the codebase with operator VI publicly available? I've been looking, but couldn't find it.
